dhis-2/dhis-api/src/main/java/org/hisp/dhis/relationship/RelationshipType.java

```diff
@@ -59,6 +59,8 @@
 
     private String toFromName;
 
+    private boolean referral = false;
```

Comment:
I think this can be removed as false is the default anyway.

Reply:
Fixed